onadata/apps/main/tests/test_csv_export.py

```diff
@@ -183,3 +183,110 @@ def test_csv_repeat_with_note(self):
                 "_media_all_received",
             ]
             self.assertEqual(sorted(expected_headers), sorted(actual_headers))
+
+    def test_csv_repeat_with_selects(self):
+        """
+        Test that select one and select multiple in a repeat are exported correctly.
+        """
+        md = """
+        | survey   |
+        |          |
+        |          | type                                | name               | label                            |
+        |          | text                                | name               | 1. What is your name?            |
+        |          | integer                             | age                | 2. How old are you?              |
+        |          | image                               | picture            | 3. May I take your picture?      |
+        |          | select one from yes_no              | has_children       | 4. Do you have any children?     |
+        |          | begin repeat                        | children           | 5. Children                      |
+        |          | text                                | childs_name        | 5.1 Childs name?                 |
+        |          | integer                             | childs_age         | 5.2 Childs age?                  |
+        |          | select one from yes_no              | child_is_nursing   | 5.3 Is the child nursing?        |
+        |          | end repeat                          |                    |                                  |
+        |          | geopoint                            | gps                | 6. Record your GPS coordinates.  |
+        |          | select all that apply from browsers | web_browsers       | 7. What web browsers do you use? |
```

Comment:
```suggestion
        |          | select_multiple  browsers          | web_browsers       | 7. What web browsers do you use? |
```

Reply:
Both syntaxes are supported. The tutorial_w_repeat.xlsx fixture from which this form was adopted has that, and we have no failures in transforming the form to XForm.